tests/test_server_plugin_integral_all_sky.py

```diff
@@ -1,3 +1,5 @@
+#TODO: this was here to make an example of dispatcher interface, but the interface is now fixed by other tests
+
```

Comment:
Does this mean that is no longer necessary?

Reply:
it will not be necessary once moved